src/Features/Core/Portable/DocumentSymbols/IDocumentSymbolsService.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.DocumentSymbols
+{
+    public interface IDocumentSymbolsService : ILanguageService
```

Comment:
let's start with this internal.  we'll want to take this through design review.  i'm very wary of htis in particular as it's a public API for all languages but exposes C#/VBisms (like ISymbols).